src/main/kotlin/uk/gov/gdx/datashare/models/CognitoClientResponse.kt

```diff
@@ -2,14 +2,14 @@ package uk.gov.gdx.datashare.models
 
 import com.fasterxml.jackson.annotation.JsonInclude
 import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.gdx.datashare.enums.RegExConstants.ACQUIRER_NAME_REGEX
 import uk.gov.gdx.datashare.enums.RegExConstants.CLIENT_ID_REGEX
-import uk.gov.gdx.datashare.enums.RegExConstants.CLIENT_NAME_REGEX
 import uk.gov.gdx.datashare.enums.RegExConstants.CLIENT_SECRET_REGEX
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @Schema(description = "Cognito Client Response")
 data class CognitoClientResponse(
-  @Schema(description = "Cognito client name", required = true, example = "hmpo", maxLength = 80, pattern = CLIENT_NAME_REGEX)
+  @Schema(description = "Cognito client name", required = true, example = "hmpo", maxLength = 80, pattern = ACQUIRER_NAME_REGEX)
```

Comment:
Is this ever the SUPPLIER? I think it's _fine_ but may confuse matters

Reply:
Yeah good catch